docs/csharp/language-reference/operators/index.md

```diff
@@ -37,7 +37,8 @@ The following table lists the C# operators starting with the highest precedence
 | Operators | Category or name |
 | --------- | ---------------- |
 | [x.y](member-access-operators.md#member-access-expression-), [f(x)](member-access-operators.md#invocation-expression-), [a&#91;i&#93;](member-access-operators.md#indexer-operator-), [x++](arithmetic-operators.md#increment-operator-), [x--](arithmetic-operators.md#decrement-operator---), [x!](null-forgiving.md), [new](new-operator.md), [typeof](type-testing-and-cast.md#typeof-operator), [checked](../keywords/checked.md), [unchecked](../keywords/unchecked.md), [default](default.md), [nameof](nameof.md), [delegate](delegate-operator.md), [sizeof](sizeof.md), [stackalloc](stackalloc.md), [x->y](pointer-related-operators.md#pointer-member-access-operator--) | Primary |
-| [+x](arithmetic-operators.md#unary-plus-and-minus-operators), [-x](arithmetic-operators.md#unary-plus-and-minus-operators), [\!x](boolean-logical-operators.md#logical-negation-operator-), [~x](bitwise-and-shift-operators.md#bitwise-complement-operator-), [++x](arithmetic-operators.md#increment-operator-), [--x](arithmetic-operators.md#decrement-operator---), [^x](member-access-operators.md#index-from-end-operator-), [x?.y](member-access-operators.md#null-conditional-operators--and-), [x?[y]](member-access-operators.md#null-conditional-operators--and-), [(T)x](type-testing-and-cast.md#cast-expression), [await](await.md), [&x](pointer-related-operators.md#address-of-operator-), [*x](pointer-related-operators.md#pointer-indirection-operator-), [true and false](true-false-operators.md) | Unary |
+| `x?.y`, `x?[y]` | [Null-conditional operators](member-access-operators.md#null-conditional-operators--and-) |
```

Comment:
@BillWagner [the more I think](https://github.com/dotnet/docs/issues/17988#issuecomment-618819640), the more I see that the following three solutions might make sense for the precedence table:
- bring `?.` and `?[]` back to the first row (pro: it's the same precedence as they are unary post-fix; con: null-conditional operators are not primary (however, we can change the name of the first row from "primary" to "primary and null-conditional")
- place `?.` and `?[]` into the separate row as the current PR does
- remove `?.` and `?[]` from the precedence table at all (for example, the spec doesn't mention the precedence of `?.` and `?[]` explicitly)

What do you think?

Reply:
Placing just `?.` and `?[]` in a separate row is not correct. Option 1 is clearly better as it does not introduce more confusion.

Reply:
@BillWagner I've updated the table as [per comment](https://github.com/dotnet/docs/issues/17988#issuecomment-620310292)